pkg/controller/clusterconnection/clusterconnection_controller.go

```diff
@@ -292,22 +295,22 @@ func (r *ReconcileConnection) Reconcile(ctx context.Context, request reconcile.R
 	}
 
 	// Validate that the tier watch is ready before querying the tier to ensure we utilize the cache.
-	if !r.tierWatchReady.IsReady() {
+	if !r.tierWatchReady.IsReady() && isCalicoEnterprise {
 		r.status.SetDegraded(operatorv1.ResourceNotReady, "Waiting for Tier watch to be established", nil, reqLogger)
 		return reconcile.Result{RequeueAfter: 10 * time.Second}, nil
 	}
 
 	// Ensure the allow-tigera tier exists, before rendering any network policies within it.
 	includeV3NetworkPolicy := false
-	if err := r.Client.Get(ctx, client.ObjectKey{Name: networkpolicy.TigeraComponentTierName}, &v3.Tier{}); err != nil {
+	if err := r.Client.Get(ctx, client.ObjectKey{Name: networkpolicy.TigeraComponentTierName}, &v3.Tier{}); err != nil && isCalicoEnterprise {
```

Comment:
We actually need to put this within an if branch that prevents the `Get` call from being made at all. Otherwise, we will be sending unnecessary requests to the Kubernetes API server.

Normally, we block on the `tierWatchReady` - this ensures that requests made with the client are served from our local cache and don't actually generate API requests. But without that, this is going to generate a lot of requests that otherwise would not have happened.